content/history.md

```diff
@@ -4,6 +4,44 @@ layout: single
 title: History
 ---
 
+### Added
```

Comment:
The heading with the version is missing here.

Reply:
I saw it 30 seconds ago :grinning: